modules/managing-memcached-operator-using-olm.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+//
+// * operators/osdk-getting-started.adoc
+
+[id='managing-memcached-oeprator-using-olm_{context}']
+= Managing a Memcached Operator using the Operator Lifecycle Manager
+
+The previous section has covered manually running an Operator. In the next
+sections, we will explore using the Operator Lifecycle Manager which is what
+enables a more robust deployment model for Operators being run in production
+environments.
+
+The Operator Lifecycle Manager helps you to install, update, and generally
+manage the lifecycle of all of the Operators (and their associated services) on
+a Kubernetes cluster. It runs as an Kubernetes extension and lets you use
+`kubectl` for all the lifecycle management functions without any additional
+tools.
+
+.Prerequisites
+
+- Operator Lifecycle Manager (OLM) installed on a Kubernetes-based cluster (v1.8
+or above to support the `apps/v1beta2` API group), for example {product-title}
+3.11 with Technology Preview OLM enabled
+- Memcached Operator built
+
+.Procedure
+
+. *Generate an Operator manifest.*
```

Comment:
Could link to this in the section if you wanted: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md

Reply:
@adellape @robszumski - we wanna avoid linking to github due to support issues. Is it possible to bring this csv file in within this doc?